core-processor/src/lib.rs

```diff
@@ -54,6 +54,7 @@ pub use executor::{calculate_gas_for_code, calculate_gas_for_program, execute_wa
 pub use ext::{Ext, ProcessorContext, ProcessorError, ProcessorExt};
 pub use handler::handle_journal;
 pub use processor::{
-    precharge, prepare, process, PrechargeResult, PrepareResult, PreparedMessageExecutionContext,
-    ProcessExecutionContext,
+    precharge_for_code, precharge_for_code_length, precharge_for_instrumentation,
```

Comment:
due to such massive usage of `ContextOfSomething`, I woould add module and type clarification docs